src/motor.c

```diff
@@ -478,13 +478,23 @@ void TIM1_CAP_COM_IRQHandler(void) __interrupt(TIM1_CAP_COM_IRQHANDLER)
   while (!(ADC1->CSR & ADC1_FLAG_EOC)) ;
   ui16_g_adc_battery_current = UI16_ADC_10_BIT_BATTERY_CURRENT;
 
+  // we ignore low values of the battery current < 5 to avoid issues with other consumers than the motor
+  // Piecewise linear is better than a step, to avoid limit cycles.
+  // in     --> out
+  // 0 -  5 --> 0 - 0
+  // 5 - 15 --> 0 - 15
+  if (ui16_g_adc_battery_current <= 5)
+    ui16_g_adc_battery_current = 0;
+  else if (ui16_g_adc_battery_current <= 15)
+    ui16_g_adc_battery_current = (ui16_g_adc_battery_current + ui16_g_adc_battery_current >> 1) - 5;
+    
```

Comment:
Used other pull request as a base, updating variables to be consistent with V1.0 code.  
![image](https://user-images.githubusercontent.com/71988810/94376245-3fe27200-00ce-11eb-93ee-472b9290b73e.png)